packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -22,6 +23,15 @@ export function toCheckpointFileName (scriptName: string): string {
   return path.join(artifactsPath, scriptName + checkpointFileSuffix);
 }
 
+export function toScriptFileName (filename: string): string {
+  filename = filename.replace(artifactsPath + path.sep, '');
+  if (filename.endsWith(checkpointFileSuffix)) {
```

Comment:
Actually it's not even needed to check this one. It's always true.

Reply:
yes